GSA-TTS/FAC
# 1848 cross validation make sure all award references in federal awards are unique

## Description
#1848 

To test locally, run:
`make docker-clean`
`make docker-first-run`
`docker compose up`
make sure backend-web is running properly and run this cmd in another terminal run
`docker compose run web python manage.py test audit.cross_validation.test_check_award_reference_uniqueness`

## PR checklist: submitters

- [ ]   Do manual testing locally. Our tests are not good enough yet to allow us to skip this step.
- [ ]   If Git surgery is necessary at any point, repeat the testing after it’s finished.

## PR checklist: reviewers

- [ ]   If Git surgery is necessary at any point (such as during a merge party), repeat the testing after it’s finished.
        
The larger the PR, the stricter we should be about these points.
